### PR TITLE
added addon configuration

### PR DESCRIPTION
**Pull request by maxwondercorn**
added configuration from old guides with some cleanup

### CODE REVIEW

File: guides/writing-addons/intro-tutorial.md [hunk 1]
```diff
@@ -290,4 +290,31 @@ Lastly, be sure to provide a few notes about how others can contribute to the pr
 
 ## Advanced addon configuration
 
-<!-- Help wanted! -->
+Addons are configured using the `ember-addon` hash in the `package.json` file.  
+
+
+
```
Comment (jenweber) [inline guides/writing-addons/intro-tutorial.md]: Please use just one line of whitespace before and after code blocks

File: guides/writing-addons/intro-tutorial.md [hunk 2]
```diff
@@ -290,4 +290,31 @@ Lastly, be sure to provide a few notes about how others can contribute to the pr
 
 ## Advanced addon configuration
```
Comment (jenweber) [inline guides/writing-addons/intro-tutorial.md]:
```suggestion
## Common addon configurations
```

Eventually I want to dedicate an entire page to addon configurations, so maybe we include solely the basics here. What do you think?

File: guides/writing-addons/intro-tutorial.md [hunk 3]
```diff
@@ -290,4 +290,31 @@ Lastly, be sure to provide a few notes about how others can contribute to the pr
...
+```
+By default, the `configPath` property is defined to point to the config directory of the test dummy application.  All other properties are optional.
+
+### before and after
+These properties specify whether your ember-addon must run "before" or "after" any other Ember CLI addons. Both of these properties can take either a string or an array of strings. The string is the name of another Ember CLI addon, as defined in the `package.json` of the other addon.
+
+### defaultBlueprint
+Addons have a default blueprint that will automatically run when the addon is installed.  By convention, Ember will run the blueprint named after the `name` property in `package.json` 
+
+You may specify a different name using `defaultBlueprint`. See the [addon blueprints](http://localhost:4200/release/advanced-use/blueprints/#addonblueprints) for more information on the default blueprint.
```
Comment (jenweber) [inline guides/writing-addons/intro-tutorial.md]:
```suggestion
You may specify a different name using `defaultBlueprint`. See the [addon blueprints](../../advanced-use/blueprints/#addonblueprints) for more information on the default blueprint.
```

I think that's the way the link should go, but please test it

File: guides/writing-addons/intro-tutorial.md (hunk 2 above)
Comment (maxwondercorn) [reply guides/writing-addons/intro-tutorial.md]: I agree

File: guides/writing-addons/intro-tutorial.md (hunk 3 above)
Comment (mansona) [reply guides/writing-addons/intro-tutorial.md]: I tested this and it works on the deploy preview 🎉

**Comment by mansona** [comment]
Thanks @maxwondercorn 🎉

**Pull request merged**